packages/firebase_auth/firebase_auth_web/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: firebase_auth_web
 description: The web implementation of firebase_auth
 homepage: https://github.com/FirebaseExtended/flutterfire/tree/master/packages/firebase_auth/firebase_auth_web
-version: 0.1.1+3
+version: 0.1.2
```

Comment:
Since you didn't add any new API surface area I think this should be 0.1.1+1

Reply:
Yep, can do!